.travis.yml

```diff
@@ -27,48 +25,57 @@ env:
-
 cache:
   pip: true
   directories:
-    - "$HOME/.npm"
-
+  - "$HOME/.npm"
 branches:
   only:
-    - master
-
+  - master
 matrix:
   include:
-    - name: "Python tests"
-      install: ./travis-scripts/python-install.sh
-      before_script:
-        - psql -c 'create database network;' -U postgres
-      script:
-        - pipenv run flake8 tasks.py network-api/
-        - pipenv run coverage run --source './network-api/networkapi' network-api/manage.py test networkapi
-      after_success:
-        - coveralls
-    - name: "npm tests"
-      before_install:
-        - nvm install --lts=carbon
-        - nvm use --lts=carbon
-      install: ./travis-scripts/npm-install.sh
-      script: npm test
-    - name: "Percy tests"
-      if: NOT TRAVIS_PULL_REQUEST_BRANCH =~ /^(dependabot\/)/
-      before_install:
-        - nvm install --lts=carbon
-        - nvm use --lts=carbon
-      install:
-        - ./travis-scripts/python-install.sh
-        - ./travis-scripts/npm-install.sh
-      before_script:
-        - psql -c 'create database network;' -U postgres
-        - npm run build
-        - pipenv run python network-api/manage.py collectstatic --no-input --verbosity 0
-        - pipenv run python network-api/manage.py migrate --no-input
-        - pipenv run python network-api/manage.py block_inventory
-        - pipenv run python network-api/manage.py sync_page_translation_fields
-        - pipenv run python network-api/manage.py update_translation_fields
-        - pipenv run python network-api/manage.py load_fake_data
-      script: npm run percy
+  - name: Python tests
+    install: "./travis-scripts/python-install.sh"
+    before_script:
+    - psql -c 'create database network;' -U postgres
+    script:
+    - pipenv run flake8 tasks.py network-api/
+    - pipenv run coverage run --source './network-api/networkapi' network-api/manage.py
+      test networkapi
+    after_success:
+    - coveralls
+  - name: npm tests
+    before_install:
+    - nvm install --lts=carbon
+    - nvm use --lts=carbon
+    install: "./travis-scripts/npm-install.sh"
+    script: npm test
+  - name: Percy tests
+    if: NOT TRAVIS_PULL_REQUEST_BRANCH =~ /^(dependabot\/)/
+    before_install:
+    - nvm install --lts=carbon
+    - nvm use --lts=carbon
+    install:
+    - "./travis-scripts/python-install.sh"
+    - "./travis-scripts/npm-install.sh"
+    before_script:
+    - psql -c 'create database network;' -U postgres
+    - npm run build
+    - pipenv run python network-api/manage.py collectstatic --no-input --verbosity
+      0
+    - pipenv run python network-api/manage.py migrate --no-input
+    - pipenv run python network-api/manage.py block_inventory
+    - pipenv run python network-api/manage.py sync_page_translation_fields
+    - pipenv run python network-api/manage.py update_translation_fields
+    - pipenv run python network-api/manage.py load_fake_data
+    script: npm run percy
+deploy:
```

Comment:
Travis docs mention something about adding the AWS region for S3 hosted site: https://docs.travis-ci.com/user/deployment/s3/#deploy-to-a-s3-hosted-website
Do we need it?

Reply:
yeah lets be explicit about region